src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,118 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private Object[] elementData = new Object [DEFAULT_CAPACITY];
+    private int size = 0;
```

Comment:
```suggestion
    private int size ;
```
class fields have default values, default value of int is 0 anyways